wled00/wled.h

```diff
@@ -769,6 +769,9 @@ WLED_GLOBAL volatile uint8_t jsonBufferLock _INIT(0);
 #define W(c) (byte((c) >> 24))
 
 class WLED {
+private:
```

Comment:
Does it make sense to have these methods also without BLE usermod? 
If not, please put them between `#ifdef USERMOD_BLE_2_JSON` ... `#endif`

Reply:
well... it can if there are other usermods that want to be able to disable wifi during the "setup" phase of booting. I certainly don't mind putting them in an `#ifdef`, though